int32[] Quality_val
float32[] sensor_val
float32[] min_range_val
float32[] max_range_val
float32[] range_val

int32  Quality_val_count
int32  sensor_val_count
int32  min_range_val_count
int32  max_range_val_count
int32  range_val_count
